modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -4492,6 +4489,20 @@ private ObjectEntry _addObjectEntry(AccountEntry accountEntry)
 			ObjectDefinitionConstants.SCOPE_COMPANY);
 	}
 
+	private ObjectEntry _addObjectEntry(
+			DTOConverterContext dtoConverterContext,
+			ObjectDefinition objectDefinition, ObjectEntry objectEntry,
+			String scope)
+		throws Exception {
+
+		ObjectEntry newObjectEntry = _defaultObjectEntryManager.addObjectEntry(
+			dtoConverterContext, objectDefinition, objectEntry, scope);
+
+		assertEquals(newObjectEntry, objectEntry);
```

Comment:
@MarinhoFeliphe This way we're performing an `assertEquals` for every test (even if they're not supposed to test it). Test should be as self-contained as possible. If something in this matter fails, every test that adds an entry will fail. 

Please, refact it in a way that we'll have a test dedicated to it.

Reply:
Also, we should have tests covering cases where we don't fill some field (integer, etc) in the request. Is there any test for it?

Reply:
Agreed!

@gabrielwas there are many changes in the tests but I've only divided the testAddObjectEntry into two (testAddObjectEntryWithStaticValues and testAddObjectEntryWithDynamicValues)